templates/beta/contracts/Imports.sol

```diff
@@ -1,10 +0,0 @@
-pragma solidity 0.4.18;
-
-// HACK to workaround truffle artifact loading on dependencies
-
-import "@aragon/os/contracts/factory/ENSFactory.sol";
-import "@aragon/os/contracts/factory/APMRegistryFactory.sol";
-import "@aragon/os/contracts/factory/EVMScriptRegistryFactory.sol";
-import "@aragon/os/contracts/common/EtherToken.sol";
-
-contract Imports {}
```

Comment:
Is this workaround not needed anymore?

Reply:
I guess so. I removed it because `solium` was complaining for having an empty contract here.

Reply:
If it's working, it'll be awesome to get rid of these hacks. Otherwise, we can ask solium to ignore this file.

Reply:
Tests were passing without it.